mLRS/Common/hal/esp-glue.h

```diff
@@ -18,16 +18,32 @@
 #endif
 
 
+// ESP32 doesn't support
 void __disable_irq(void)
 {
+#ifdef ESP8266
+    noInterrupts();
+#endif
+}
+
+// ESP32 doesn't support
+void __enable_irq(void)
+{
+#ifdef ESP8266
+    interrupts();
+#endif
+}
+
+void enterCritical(void)
+{
 #ifdef ESP32
     taskENTER_CRITICAL(&esp32_spinlock);
 #elif defined ESP8266
     noInterrupts();
 #endif
 }
 
-void __enable_irq(void)
+void exitCritical(void)
```

Comment:
that's in principle not a bad idea, but I would suggest that we just don't try for now to do anything smart. That is, that we don't define any such functions, keep __enable_irq()/__disable_irq() empty, and instead code what is needed at the places where it is needed
will not look super nice for now, but at least allows us to do what it is needed where something is needed exactly as it is needed
the  __enable_irq()/__disable_irq() can also interfer with tx stm32 code, so let's keep them empty

Reply:
Should be addressed now